apps/authentication-portal/src/main/webapp/emailOtp.jsp

```diff
@@ -116,12 +116,18 @@
                         <label for="password"><%=AuthenticationEndpointUtil.i18n(resourceBundle, "enter.code")%>
                             (<%=Encode.forHtmlContent(request.getParameter("screenValue"))%>)
                         </label>
-                        <input type="password" id='OTPCode' name="OTPCode" c size='30'/>
+                        <div class="ui fluid icon input addon-wrapper">
+                            <input type="password" id='OTPCode' name="OTPCode" c size='30'/>
+                            <i id="password-eye" class="eye icon right-align password-toggle slash" onclick="showOTPCode()"></i>
+                        </div>
                             <% } else { %>
                         <div class="field">
                             <label for="password"><%=AuthenticationEndpointUtil.i18n(resourceBundle, "enter.code")%>
                                 :</label>
-                            <input type="password" id='OTPCode' name="OTPCode" size='30'/>
+                            <div class="ui fluid icon input addon-wrapper">
+                                <input type="password" id='OTPCode' name="OTPCode" size='30'/>
+                                <i id="password-eye" class="eye icon right-align password-toggle slash" onclick="showOTPCode()"></i>
```

Comment:
I think the icon should also change when toggled.

Reply:
Fixed